mobile-widgets/qml/About.qml

```diff
@@ -21,16 +21,16 @@ Kirigami.ScrollablePage {
 			Layout.topMargin: Kirigami.Units.gridUnit
 			Layout.alignment: Qt.AlignHCenter
 			Layout.maximumWidth: pageWidth
-			wrapMode: TextEdit.WrapAtWordBoundaryOrAnywhere
+			wrapMode: TextEdit.NoWrap
+			fontSizeMode: Text.Fit
 		}
 		Image {
 			id: image
 			source: "qrc:/qml/subsurface-mobile-icon.png"
-			width: pageWidth / 2
-			height: width
-			fillMode: Image.Stretch
-			Layout.alignment: Qt.AlignCenter
-			horizontalAlignment: Image.AlignHCenter
+			fillMode: Image.PreserveAspectCrop
+			Layout.alignment: Qt.AlignHCenter
+			Layout.maximumWidth: pageWidth / 2
+			Layout.maximumHeight: Layout.maximumWidth
```

Comment:
would this work on both landscape and portrait mode?

http://doc.qt.io/qt-5/qml-qtquick-image.html#fillMode-prop
why not do `PreserveAspectFit`?

Reply:
extra points for including screenshots :+1:

Reply:
> ou got a point with landscape, I will look into that (I hope kirigami can allow another layout for landscape).

> If I understand it correctly (and saw how it looked when trying) the difference between
PreserveAspectFit and PreserveAspectCrop is that "Fit" will stretch if needed while crop
will reduce while needed.

i see,
something in these lines might work (untested):
```
property int pageWidth: aboutPage.width - aboutPage.leftPadding - aboutPage.rightPadding
property int pageHeight: aboutPage.height - aboutPage.topPadding - aboutPage.bottomPadding

...

fillMode: Image.PreserveAspectCrop
Layout.alignment: Qt.AlignHCenter
Layout.maximumWidth: pageWidth < pageHeight ? pageWidth / 2 : pageHieght / 2
Layout.maximumHeight: Layout.maximumWidth
```

the contents of the second image are not centered horizontally, not sure if that was the case before and what is causing it.